### ISSUE TITLE
Default warning color asset does not have high enough contrast

### ISSUE THREAD

**Issue by autoreleasefool** [autoreleasefool/approach#266]
As per title, contrast is too low for proper readability

<img width="1137" alt="Screenshot 2023-07-09 at 9 35 18 PM" src="https://github.com/autoreleasefool/approach/assets/6619581/b2dd0007-c152-4ca2-abec-a946a2433c86">
